writeKML.js

```diff
@@ -64,16 +171,17 @@ module.exports = {
         // Loop through nodes list, generate KML Placemark for each
         update.nodes.forEach(node => {
 
-            // Get band string from first digit of frequency field
-            const freq1 = node.meshrf && String(node.meshrf.freq)[0] || 'None';
-            const band_name = BAND[freq1] || 'Other';                
+            // Get band from 1st digit of frequency field, generate band string & styleUrl
+            const freq1 = node.meshrf && String(node.meshrf.freq)[0] || 'N';
+            const band_name = BAND[freq1] || 'Other';
+            const styleUrl = '#sm_nodes_' + BAND[freq1] || '#sm_nodes'
```

Comment:
Do you mean
```const styleUrl = '#sm_nodes_' + (BAND[freq1] || '#sm_nodes');```
or something else? Are you trying to provide a default style if BAND[freq1] doesn't exist? (which can happen)

Reply:
Yes, the goal is to provide a default if there's no band.  I already detect "None", and set the band to "NoRF"... but now realize that's probably not the right thing to do.  The style URL should be either something like: #sm_nodes_5ghz, OR just #sm_nodes (as a default).  I don't think your parentheses do that, and instead it should be: 
`const styleUrl = ('#sm_nodes_' + BAND[freq1]) || '#sm_nodes';`
That said, I'm going to rework it a bit.  Stand by for updated PR.

Reply:
Updated code.  No longer using "NoRF", but rather just "None" in both "band_name" and "as part of the "styleUrl". Added some error detection to catch "Other" situations and use default "#sm_nodes" styleUrl.